packages/web-twig/src/Resources/components/FileUploader/FileUploader.twig

```diff
@@ -0,0 +1,20 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'FileUploader' -%}
+
+{# Miscellaneous #}
+{%- set _classNames = [ _rootClassName, _class ] -%}
+
+<div
+    {{ mainProps(props) }}
+    {{ classProp(_classNames) }}
+    data-toggle="fileUploader"
```

Comment:
Please, remove this line. This must be added by the user/developer.

Reply:
OK

Reply:
Still not removed.